align/cell_fabric/grid.py

```diff
@@ -119,8 +119,8 @@ def __init__( self, *, colors, pitch, width, offset=0, repeat=None):
         self.semantic()
 
 class EnclosureGrid(Grid):
-    def __init__( self, *, clg=None, pitch, offset=0, stoppoint, check=False):
-        assert not check or 2*stoppoint <= pitch
+    def __init__( self, *, clg=None, pitch, offset=0, stoppoint, check=True):
+        # assert not check or 2*stoppoint <= pitch
```

Comment:
Why is this assertion needed?

Reply:
To prevent accidental shorts I believe. Without this condition, a wire ending at (0, 1) will always be shorted with a wire starting at (1, -1). Basically the physical coordinate of (0, 1) will be less than (1, -1) if this condition is violated (see lines 126 to 130). It is possible our higher level logic is not exercising this corner case of course.

Let's see whether circleci complaints.